packages/amplify_datastore/example/android/app/build.gradle

```diff
@@ -60,4 +61,9 @@ flutter {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```

Comment:
This was removed in other gradle files - needed here?

Reply:
Oh darn missed scanning on the example Apps 😅 thanks I will add a commit to cleaning this up